core/dbt/include/global_project/macros/etc/get_relation_comment.sql

```diff
@@ -26,3 +26,19 @@
   {% endif %}
 
 {% endmacro %}
+
+
+{% macro get_relation_column_comments(persist_docs, model) %}
+
+  {%- if persist_docs is not mapping -%}
+    {{ exceptions.raise_compiler_error("Invalid value provided for 'persist_docs'. Expected dict but got value: " ~ persist_docs) }}
+  {% endif %}
+
+  {% if persist_docs.get('columns', false) and model.columns|length != 0 %}
+    {{ return(model.columns) }}
+  {%- else -%}
+    {{ return(none) }}
+  {% endif %}
+
+{% endmacro %}
```

Comment:
The same as my above comment applies here - I think these two are the only overlap!